src/firefly/js/visualize/draw/DrawUtil.js

```diff
@@ -126,7 +127,9 @@ function drawText(drawTextAry,text, x,y,color,
         fontStyle,
         'MozBorderRadius': '1px',
         'borderRadius': '1px',
-        'WebkitBorderRadius': '1px'
+        'WebkitBorderRadius': '1px',
+        transform: 'rotate('+rotationAngle+')'
```

Comment:
pass as undefined as the default and then if defined add to to the style like `backGroundColor` on line 135.   This way you don't add unnecessary stuff to the style.